apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -0,0 +1,247 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { type CourseRequestRow } from './instructorRequestCourse';
+
+export const RequestCourse = ({
+  course_requests,
+  resLocals,
+}: {
+  course_requests: CourseRequestRow[];
+  resLocals: Record<string, any>;
+}) => {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head')%>", {
+          ...resLocals,
+        })}
+        <script>
+          $(function () {
+            $('input[name=cr-role]').change(function () {
+              var role = this.value;
+              $('.question-form button').prop('disabled', role != 'instructor');
+              $('.role-comment').hide();
+              $('.role-comment-' + role).show();
+            });
+          });
+        </script>
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../partials/navbar')%>", {
+          ...resLocals,
+          navPage: 'request_course',
+        })}
+        <main id="content" class="container">
+          ${course_requests.length > 0
+            ? html`
+                <!-- Previous Course Requests -->
+                <div class="card mb-4">
+                  <div class="card-header bg-primary text-white d-flex align-items-center">
+                    Course Requests
+                  </div>
+                  <div class="table-responsive">
+                    <table class="table table-sm table-hover table-striped">
+                      <thead>
+                        <tr>
+                          <th>Short Name</th>
+                          <th>Title</th>
+                          <th>Status</th>
+                          <th>Details</th>
+                        </tr>
+                      </thead>
+                      <tbody>
+                        ${course_requests.map((req) => {
```

Comment:
Use destructuring here:
```suggestion
                        ${rows(({ course_request, approved_by_user }) => {
```
This avoids the need to ever refer to the whole row variable.

Reply:
I presume you mean `rows.map(({ course_request, approved_by_user }).....`?